### PR TITLE
Fixed cmake generation warning for regeneration project build files.

### PR DESCRIPTION
**Pull request by proydakov**
Hi, GLSLANG Team.

Then I'm trying to regenerate project build files I see cmake warning.
As I see from kitware cmake main repo we should use 'cmake_minimum_version' before 'project' directive:

https://github.com/Kitware/CMake/blob/master/CMakeLists.txt

Could you verify and accept my changes, pls?

[BEFORE]:

% cmake ..
CMake Warning (dev) at CMakeLists.txt:35 (project):
  Policy CMP0048 is not set: project() command manages VERSION variables.
  Run "cmake --help-policy CMP0048" for policy details.  Use the cmake_policy
  command to set the policy and suppress this warning.

  The following variable(s) would be set to empty:

    CMAKE_PROJECT_VERSION
    CMAKE_PROJECT_VERSION_MAJOR
    CMAKE_PROJECT_VERSION_MINOR
    CMAKE_PROJECT_VERSION_PATCH
This warning is for project developers.  Use -Wno-dev to suppress it.

-- No build type selected, default to Debug
-- Found PythonInterp: /usr/local/bin/python3 (found version "3.9")
-- Found PythonInterp: /usr/local/bin/python3 (found suitable version "3.9", minimum required is "3")
-- optimizer enabled
-- Google Mock found - building tests
-- Configuring done

[AFTER]:

% cmake ..
-- No build type selected, default to Debug
-- Google Mock was not found - tests based on that will not build
-- Configuring done
-- Generating done

Best regards, Proydakov Evgeny

### CODE REVIEW

**Comment by proydakov** [comment]
As I see all CI passed without problems. Is it possible to accept my PR, pls?

File: CMakeLists.txt [hunk 1]
```diff
@@ -31,15 +31,15 @@
 # ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 
-project(glslang
-    LANGUAGES CXX)
-
+cmake_minimum_required(VERSION 2.8.12)
 # increase to 3.1 once all major distributions
 # include a version of CMake >= 3.1
```
Comment (greg-lunarg) [inline CMakeLists.txt:36]: Can you please move these two comment lines above the cmake_minimum_required() command?

File: CMakeLists.txt [hunk 2]
```diff
@@ -31,15 +31,15 @@
 # ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 
-project(glslang
-    LANGUAGES CXX)
-
+cmake_minimum_required(VERSION 2.8.12)
```
Comment (greg-lunarg) [inline CMakeLists.txt:34]: Can you please also move the two comment lines below to above the cmake_minimum_required() command? They go with this command.

**Review by greg-lunarg — CHANGES_REQUESTED** [review]
Thank you for fixing this. Just a simple comment change is needed for check in.

File: CMakeLists.txt (hunk 2 above)
Comment (proydakov) [reply CMakeLists.txt:34]: Yep.

**Pull request merged**